packages/client/hmi-client/src/services/document-assets.ts

```diff
@@ -2,12 +2,10 @@
  * Documents Asset
  */
 
-import API, { PollerState } from '@/api/api';
-import type { AddDocumentAssetFromXDDResponse, Document, DocumentAsset } from '@/types/Types';
+import API from '@/api/api';
+import type { Document, DocumentAsset } from '@/types/Types';
```

Comment:
Can we now fully delete `AddDocumentAssetFromXDDResponse.java` and have it taken out of Types.ts too?

Reply:
Let's wait.